### PR TITLE
[Feature] Fractus Nodes

### PR DESCRIPTION
**Pull request by Hiroshi97**

### CODE REVIEW

**Comment by Hiroshi97** [comment]
@2ndtlmining Hi, just to clarify: **Fractus** is also Cumulus nodes with Thunder is false, is that correct? If so, some of data must be pending to calculate and get the Fractus from total Cumulus nodes. 

Regarding to the header, the API returns only the total of cumulus, stratus, and nimbus so we don't have any data about Fractus yet in that time to display.

**Comment by 2ndtlmining** [comment]
Thanks ill have a test and see how it goes. I dont own one of these so ill have to ask around for some help to test.

Your question: Fractus is also Cumulus nodes with Thunder is False= Yes and No :).  Fractus is a Cumulus node thus will sit within the same queue for payment but it has the Thunder flag set to True. Its basically a storage node in the Cumulus bucket. There is a 10TB requirement a bit more upload speed. So its just a different version of Cumulus if that makes sense.

The project to introduce these types of nodes was called thunder and as such made its way into the API and code.

Cool, if the API returns only Cumulus then we leave that part as is till i can see if there is an API for it. Ill ask the question so long. Let you know how testing goes

**Comment by 2ndtlmining** [comment]
Hey mate

Managed to find a wallet:

image


Couple things:

1. DWS Validation: For cractus needs to be 80mbs 
2. Icon in the Payout timer think still going to Cumulus
3. Was looking at the code changes and spotted the colateral being 0, that is 1000 but not think its actually used. 

I am still doing some other testing

**Comment by 2ndtlmining** [comment]
Another small thing, in white mode think the Fractus toggle is not visible. Suspect its just white, if that cant be fixed not a big deal:

image

**Comment by Hiroshi97** [comment]
@2ndtlmining Thanks for the feedback:
1. Fixed
2. Fixed
3. I don't know what it's actually for so just set it = 0 just in case we have any other value for the FRACTUS collateral. I've set it to 1000.
4. Toggle button for Light theme: Fixed

File: client/src/main/apidata.js [hunk 1]
```diff
@@ -186,9 +200,8 @@ export async function fetch_global_stats(walletAddress = null) {
 
   if (resTotalDonations.status == 'fulfilled') {
     const res = resTotalDonations.value;
-    store.total_donations = res;
```
Comment (victorquanlam) [inline client/src/main/apidata.js:189]: @Hiroshi97 oops. You left extra quotation marks here buddy 😄

File: client/src/main/apidata.js [hunk 2]
```diff
@@ -186,9 +200,8 @@ export async function fetch_global_stats(walletAddress = null) {
...
+    store.total_donations = `res`;
```
Comment (victorquanlam) [inline client/src/main/apidata.js:203]: @Hiroshi97 oops. You left extra quotation marks here buddy 😄

**Pull request merged**